src/helpers/getDeckAdvice/advice/OBSIDIAN_BUTCHERS/index.js

```diff
@@ -9,27 +11,38 @@ export default cards => {
   const hasObsidianButchers = cardIds.includes('F17')
   const hasRainOfFrogs = cardIds.includes('F8')
   const hasAzureHatchers = cardIds.includes('F10')
+  const hasLudicMatriarchs = cardIds.includes('N43')
+  let hasLowLvlMatriarchs = false
+  if (hasLudicMatriarchs) {
+    hasLowLvlMatriarchs = cards.find( card =>  card.id === 'N43' ).level <= 2 ? true : false
+  }
+  const numDragons = getDragons(cards).length
```

Comment:
I tend to find these variables clumsy to name, so I would usually write the code like this instead:
```suggestion
  const dragons  = getDragons(cards)
```

And then use `dragons.length`. It saves me from coming up with additional terminology like “amount“, “count”, “num” or “number”.